OpenShot/openshot-qt
# Fixing Razor Cursor (which was broken on Windows & webkit backends)

Fixing razor cursor, which was broken on Windows (i.e. webkit backend) when we moved from PNG to SVG icon file. Also, adding a new way to target CSS for webkit VS webengine.

We will eventually move away from a JS/CSS approach to the timeline, so this is really a bandage for now, to allow some slight differences on how we CSS style the timeline (depending on the webkit / webengine backend used).

**Testing Instructions:**
- Launch this build on Windows (or run with `--web-backend webkit` backend)
- Drop an emoji or video on the timeline
- Click the 'Razor' toolbar icon
- Move your mouse over the clip, and you should see the razor cursor
